resources/qml/delegates/Encrypted.qml

```diff
@@ -4,45 +4,71 @@
 
 import ".."
 import QtQuick 2.15
-import QtQuick.Controls 2.1
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
 import im.nheko 1.0
 
-Column {
+Rectangle {
     id: r
 
     required property int encryptionError
     required property string eventId
 
+    radius: fontMetrics.lineSpacing + Nheko.paddingMedium
```

Comment:
I think I was wrong, it should be `fontMetrics.lineSpacing/2 + Nheko.paddingMedium`